README.md

```diff
@@ -41,7 +41,7 @@ To get a local copy up and running follow these simple example steps.
 
 ### Github page for this project
 
-- 
+- https://fernando-silvabr66.github.io/leaderboard/dist
```

Comment:
- [ ] OPTIONAL: The app screenshot is broken, please kindly fix the link. It looks like the link is broken, because the file extension is written with uppercase  [app-screenshot.JPG](https://github.com/fernando-silvabr66/leaderboard/blob/final-touch/app-screenshot.JPG) 😉

Reply:
Thanks for the suggestion. Done!